play.pokemonshowdown.com/src/client-main.ts

```diff
@@ -761,6 +761,58 @@ export class PSRoom extends PSStreamModel<Args | null> implements RoomOptions {
 				PS.join('login' as RoomID);
 			}
 			return true;
+		case 'open':
+		case 'user': {
+			let roomid = `user-${toID(target)}` as RoomID;
+			PS.join(roomid, {
+				rightPopup: true,
```

Comment:
```suggestion
```

`rightPopup` opens popups to the right of the thing you clicked, it's used for user lists. It doesn't make sense for a text command.